example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -0,0 +1,229 @@
+//
+// Example that demonstrates Offboard velocity control using LOCAL_NED & BODY_NED
+//
+// Author: Julian Oes <julian@oes.ch>, Shakthi Prashanth <shakthi.prashanth.m@intel.com>
+#include <iostream>
+#include <cmath>
+#include <thread>
+#include <chrono>
+#include <dronecore/dronecore.h>
+
+using namespace dronecore;
+
+#define ERROR_CONSOLE_TEXT "\033[31m" //Turn text on console red
+#define TELEMETRY_CONSOLE_TEXT "\033[34m" //Turn text on console blue
+#define NORMAL_CONSOLE_TEXT "\033[0m"  //Restore normal console colour
+
+void wait_msec(int msec)
+{
+    std::this_thread::sleep_for(std::chrono::milliseconds(msec));
+}
+
+void wait_sec(int sec)
+{
+    std::this_thread::sleep_for(std::chrono::seconds(sec));
+}
+
+int main(int argc, char **argv)
+{
+    DroneCore dc;
+    std::string offb_mode = (argc > 1) ? argv[1] : "";
+
+    if (argc == 1 || (offb_mode != "local-ned" && offb_mode != "body-ned")) {
+        std::cerr << "Usage: offboard <coordinate-frame>\n" <<
+                  "coordinate-frame: \"local-ned\", or \"body-ned\""
+                  << std::endl;
+        return EXIT_FAILURE;
+    }
+
+    // add udp connection
+    DroneCore::ConnectionResult ret = dc.add_udp_connection();
+    if (ret != DroneCore::ConnectionResult::SUCCESS) {
+        std::cout << ERROR_CONSOLE_TEXT << "Connection failed: "
+                  << DroneCore::connection_result_str(ret)
+                  << NORMAL_CONSOLE_TEXT << std::endl;
+        return EXIT_FAILURE;
+    }
+
+    // Wait for device to connect via heartbeat.
+    wait_sec(2);
+    Device &device = dc.device();
```

Comment:
You can now use `is_connected()`, see https://github.com/dronecore/DroneCore/blob/develop/include/dronecore.h#L161.